libs/blocks/video/video.js

```diff
@@ -1,13 +1,30 @@
 export default function init(a) {
   const { pathname, hash } = a;
-
   const isAutoplay = !!(hash?.includes('autoplay'));
   const isNotLooped = !!(hash?.includes('autoplay1'));
-
-  const attrs = isAutoplay ? `playsinline autoplay ${isNotLooped ? '' : 'loop'} muted` : 'playsinline controls';
+  const playOnHover = !!(hash.includes('hoverplay'));
+  let attrs = '';
+  switch (true) {
+    case (isAutoplay === true && isNotLooped === false):
+      attrs = 'playsinline autoplay loop muted';
+      break;
+    case (playOnHover === true && isNotLooped === true):
+      attrs = 'playsinline autoplay muted data-hoverplay';
+      break;
+    case (isNotLooped === true):
+      attrs = 'playsinline autoplay muted';
+      break;
+    default:
+      attrs = 'playsinline controls';
+  }
```

Comment:
nit: break out into a function (`getAttrs`?), change `isNotLooped` to `isLooped`

```suggestion
// Outside of init func
const getAttrs = (isAutoplay, isLooped, playOnHover) => {
  if (isAutoplay && isLooped) {
    return 'playsinline autoplay loop muted';
  }
  if (playOnHover && !isLooped) {
    return 'playsinline autoplay muted data-hoverplay';
  }
  if (!isLooped) {
    return 'playsinline autoplay muted';
  }
  return 'playsinline controls';
};

// in init
const attrs = getAttrs(...)
```

Reply:
This is done. I have passed hash as the parameter to the function getAttrs() also renamed isLooped to isAutoplayOnce for better clarity. Please let me know if that looks ok.

Reply:
I have updated the code to use if statements